libkbfs/journal_server.go

```diff
@@ -104,26 +110,26 @@ func makeJournalServer(
 	return &jServer
 }
 
-func (j *JournalServer) getDirLocked() string {
+func (j *JournalServer) rootPath() string {
+	return filepath.Join(j.dir, "v1")
+}
+
+func (j *JournalServer) tlfJournalPathLocked(tlfID TlfID) string {
 	if j.currentVerifyingKey == (VerifyingKey{}) {
 		panic("currentVerifyingKey is zero")
 	}
 
 	// Device IDs and verifying keys are globally unique, so no
 	// need to have the uid in the path. Furthermore, everything
 	// after the first two bytes (four characters) are randomly
-	// generated, so taking the first 36 characters gives ~1/2^64
-	// collision probability.
-	//
-	// TODO: Write out a file with the full string.
-	shortID := j.currentVerifyingKey.String()[0:36]
-	return filepath.Join(j.dir, "v1", shortID)
-}
-
-func (j *JournalServer) getDir() string {
-	j.lock.RLock()
-	defer j.lock.RUnlock()
-	return j.getDirLocked()
+	// generated, so taking the first 20 characters of the
```

Comment:
I thought about dropping the first four and two chars (respectively), but not sure if it's worth the annoyance (i.e., have to remember to do so when manually building TLF journal paths while debugging).